submissions/omolayoOdupe/todo_app/src/components/TodoStatus.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+const TodoStatus = (props) => {
+    return (
+        <div className='status'>
+            <span className='status1'>{props.itemsLeft().length}Items left</span>
```

Comment:
I think you should add a space between the number of items left and the text @OmolayoO